it/test/services/UserGroupSearchServiceISpec.scala

```diff
@@ -43,8 +43,8 @@ class UserGroupSearchServiceISpec extends IntegrationSpecBase {
 
       }
       val emailEncrypted: String =
-        ((sessionRepository
-          .get(request.sessionID))
+        (sessionRepository
+          .get(request.sessionID)
           .futureValue
           .get
           .data("AccountDetailsFor6902202884164548")
```

Comment:
Can we use TestITData.CREDENTIAL_ID instead of literal string here? Just make it clearer